pkgs/applications/misc/elogind/default.nix

```
@@ -47,11 +48,14 @@ stdenv.mkDerivation rec {
     gettext
     libxslt.bin # xsltproc
     docbook5 docbook_xsl docbook_xsl_ns docbook_xml_dtd_42 docbook_xml_dtd_45 # needed for docbook without Internet
-    (python3.withPackages (p: with p; [ lxml ]))  # fixes: man/meson.build:111:0: ERROR: Could not execute command "/build/source/tools/xml_helper.py".
+
+    # fixes: man/meson.build:111:0: ERROR: Could not execute command "/build/source/tools/xml_helper.py".
+    python3
```

Isn't this redundant, given that equivalent to
```suggestion
    python3Packages.python
```

?

Or  maybe it's another splicing dark magic?

---

You're right. :)